### ISSUE TITLE
Update both page title and meta desc of the following LOB samples

### ISSUE THREAD

**Issue by zdrawku** [IgniteUI/igniteui-angular-samples#2390]
- [Boston marathon](https://www.infragistics.com/angular-demos-lob/grid/grid)
- [FinJS ](https://www.infragistics.com/angular-demos-lob/finjs-sample)- page title: Real-Time Financial Analysis Data Grid
- [TreeGrid FinJS
](https://www.infragistics.com/angular-demos-lob/treegrid-finjs-sample)
- [CRM](https://staging.infragistics.com/angular-demos-grid-crm/grid-crm)
- [Data Analysis with dock manager](https://www.infragistics.com/angular-demos-lob/grid-dynamic-chart-data/data-analysis-dock-manager) - page title: Real-Time Data Analysis and Dashboards
- [Data Analysis](https://www.infragistics.com/angular-demos-lob/grid-dynamic-chart-data/data-analysis)
- [Master detail 
](https://www.infragistics.com/angular-demos-lob/grid/grid-master-detail)

All samples should have:
- page title
- meta description

@aatanasov2, could you please assist with the meta desc and page title

**Comment by aatanasov2** [comment]
@dobromirts  my recommendations here:

https://infragisticsinc297.sharepoint.com/:x:/g/Groups/marketing/EWd0DkPSOphHlXiOBmDgKXYB_niCf_TkOVUFKp4zu78s-Q?e=gFCTQ0

**Comment by zdrawku** [comment]
Reopening the issue as we need to add og:title element - @aatanasov2 is currently working on it.

For example, the og:title element provides description that is used for URL Preview:

URL:
https://github.com/IgniteUI/igniteui-angular/pull/8671

OG:
```
<meta property="og:image" content="https://avatars0.githubusercontent.com/u/5366066?s=400&amp;v=4" />
<meta property="og:site_name" content="GitHub" />
<meta property="og:type" content="object" />
<meta property="og:title" content="Add string enum for the activeView input by hanastasov · Pull Request #8671 · IgniteUI/igniteui-angular" />
<meta property="og:url" content="https://github.com/IgniteUI/igniteui-angular/pull/8671" />
<meta property="og:description" content="Closes #8660 Additional information (check all that apply)..
```


The current state of the samples is not enough as it look like the URL Preview logic is using the index.html file instead of the meta title and desc specified in the app.

https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fwww.infragistics.com%2Fangular-demos-lob%2Fgrid-dynamic-chart-data%2Fdata-analysis-dock-manager

Page source:
![image](https://user-images.githubusercontent.com/11193764/102221040-512b6580-3eea-11eb-9423-5b5f1e5c189d.png)

view-source:https://www.infragistics.com/angular-demos-lob/grid-dynamic-chart-data/data-analysis-dock-manager
